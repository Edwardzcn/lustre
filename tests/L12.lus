function test(x : int) returns ()
let
() = ();
tel
