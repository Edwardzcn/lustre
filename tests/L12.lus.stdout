node (x : int)
  returns ()
  assumes ()
  shows ()
let
tel
--- Step 1 ---
x : int  = 1
--- Step 2 ---
x : int  = 2
--- Step 3 ---
x : int  = 3
--- Step 4 ---
x : int  = (EOF)
